submissions/kris-almort/a-tiny-js-world/index.js

```diff
@@ -72,18 +72,23 @@ const catWoman = {
 };
 
 function preparePrint(obj) {
+  const { species, name, gender, legs, saying } = obj;
```

Comment:
You can also destructure your obj directly in parameters 
`function preparePrint({ species, name, gender, legs, saying } {...}`

It is up to you.

Reply:
done